gatsby-config.js

```diff
@@ -115,6 +115,16 @@ module.exports = {
               beacon: 'staging-bam.nr-data.net',
               errorBeacon: 'staging-bam.nr-data.net',
             },
+            production: {
+              instrumentationType: 'proAndSPA',
+              accountId: '10175106',
+              trustKey: '1',
+              agentID: '29883416',
+              licenseKey: '23448da482',
+              applicationID: '29883416',
+              beacon: 'staging-bam-cell.nr-data.net',
```

Comment:
Did these values change recently? I see our `development` config uses `staging-bam.nr-data.net`. I assume this works correctly?

Reply:
I noticed that as well. I took these values straight from what was provided in the NR Browser snippet, which is what the plugin [suggests](https://github.com/newrelic/gatsby-plugin-newrelic#installation) we do. 🤷‍♂️

Reply:
Oh awesome. As long as that is the case, we should be safe. Thanks!